i18n/en/contributing.md

```diff
@@ -1,5 +1,3 @@
-# Contributing
```

Comment:
Why do you want to remove this?

Reply:
Sorry for the confusion. I should have stated my reasoning in the first place.

I saw that this heading does not exist in any translations of this file (although the English version is, of course, the go-to version in case of doubt) and deemed it unnecessary because the page title "General Contribution Guidelines" already has "Contribution" in it.

What I did not see, however, and only realized after your question was that there are two other headings on the same level (#): Code of Conduct (22/C4.1) and Governance Process. Therefore, I would keep the heading now, i.e., delete the commit and rebase, and, consequentially, add it to the German "contributing.md" file.

Just to be sure, is this ok?

Reply:
Sounds great, thanks for this and all the other work, @rodolfo912

Reply:
Ok. I have done so now. I thank you for Kovri. It's a great project. I'm glad if I can help a little.